test/test_mcmc.py

```diff
@@ -39,8 +37,8 @@ def test_logistic_regression(algo):
     warmup_steps, num_samples = 1000, 8000
     data = random.normal(random.PRNGKey(0), (N, dim))
     true_coefs = np.arange(1., dim + 1.)
-    probs = expit(np.sum(true_coefs * data, axis=-1))
-    labels = dist.bernoulli(probs).rvs(random_state=random.PRNGKey(0))
+    logits = np.sum(true_coefs * data, axis=-1)
```

Comment:
This should already be in master. Could you merge master into your branch?

Reply:
The master branch only changes the code in the model, not data. Either way does not matter because under the hood, logits will be converted to probs. I make this change just for consistence with the model.